src/locales/es-ES.ts

```diff
@@ -80,19 +80,131 @@ export default {
   'app.currentpw': 'Contraseña actual',
   'app.newpw': 'Nueva Contraseña',
   'app.updateinfo': 'Actualizar Cuenta',
-  'app.minimum': 'Mí­nimo',
+  'app.minimum': 'Mí¬nimo',
```

Comment:
'app.minimum': 'Mínimo',

Reply:
agreed. I want to see if there is a faster way I can do this change without submitting another PR so I will await trigger.